bdsf/readimage.py

```diff
@@ -233,32 +233,38 @@ def gaus2pix(x, location=None, use_wcs=True):
             else:
                 return img.beam2pix(x)
 
-        def pix2gaus(x, location=None, use_wcs=True):
+        def pix2gaus(x, location=None, use_wcs=True, is_error=False):
             """ Converts Gaussian parameters in pixels to deg.
 
             x - (maj [pix], min [pix], pa [deg])
             location - specifies the location in pixels (x, y) for which
                 transform is desired
             Input beam angle should be degrees CCW from the +y axis of the image.
             The output beam angle is degrees CCW from North.
+            Set is_error = True when x gives the errors on the parameters instead of
```

Comment:
Does "gives" mean "contains" here?

Reply:
Yes. I changed it to "contains" to be clear.